CONTRIBUTORS.md

```diff
@@ -18,6 +18,7 @@
 * [helpermika](https://github.com/helpermika)
 * [Ice3man543](https://github.com/Ice3man543)
 * [JamTookTheBait](https://github.com/JamTookTheBait)
+* [justinsteven](https://github.com/justinsteven)
```

Comment:
Trying to keep the contributors list alphabetically sorted (case insensitive), can you fix to to give my OCD rest ;)

Reply:
I don't know how I managed to mess up that bad :sweat_smile: thanks